### ISSUE TITLE
Datatable Stuck on Processing

### ISSUE THREAD

**Issue by minedun6** [yajra/laravel-datatables#1085]
This seems rather odd, The request is properly made and the data is returned correctly when I check it in the Network Tab of chrome devtools.
Problem is that the view is stuck on the "preocessing" state and doesn't load the data in the table ?

I'm on version 6.27 and I want to pinpoint that till yesterday it was working correctly.

**Comment by yajra** [comment]
No error thrown at all? There must be something wrong on you javascript and table? Make sure you have the same number of columns on your table and on your javascript and table footer if any.

**Comment by minedun6** [comment]
Already sorted it out, thank you.
Btw maybe this is not the place to talk about this, regarding anyone having issues with large dataset, I can assure that it works properly, and if you have slow in the process, it's coming from your db.
How did I found out, I took the generated query from the package and ran it in the sql editor, it was also runing slow, and after some indexing, it's almost automatic ^^.
Hope this helps.
